tests/imfile-wildcards-dirs-multi3.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# This is part of the rsyslog testbench, licensed under GPLv3
+export IMFILEINPUTFILES="1"
+export IMFILEINPUTFILESSTEPS="5"
+export IMFILEINPUTFILESALL=$(($IMFILEINPUTFILES * $IMFILEINPUTFILESSTEPS))
+echo [imfile-wildcards-multi3.sh]
+. $srcdir/diag.sh check-inotify-only
+. $srcdir/diag.sh init
+# generate input files first. Note that rsyslog processes it as
+# soon as it start up (so the file should exist at that point).
+
+for i in `seq 1 $IMFILEINPUTFILES`;
+do
+	echo "Make rsyslog.input.dir$i"
+	mkdir rsyslog.input.dir$i
+done
+
+# Start rsyslog now before adding more files
+. $srcdir/diag.sh startup imfile-wildcards-dirs-multi3.conf
+# sleep a little to give rsyslog a chance to begin processing
+sleep 2
+
+for j in `seq 1 $IMFILEINPUTFILESSTEPS`;
+do
+	echo "Loop Num $j"
+	for i in `seq 1 $IMFILEINPUTFILES`;
+	do
+		echo "Make rsyslog.input.dir$i/dir$j/testdir"
+		mkdir rsyslog.input.dir$i/dir$j
+		mkdir rsyslog.input.dir$i/dir$j/testdir
+		mkdir rsyslog.input.dir$i/dir$j/testdir/subdir$j
+		./inputfilegen -m 1 > rsyslog.input.dir$i/dir$j/testdir/subdir$j/file.logfile
+	done
+	ls -d rsyslog.input.*
+
+	./msleep 250
+
+	# Delete all but first!
+	for i in `seq 1 $IMFILEINPUTFILES`;
+	do
+		rm -rf rsyslog.input.dir$i/dir$j/testdir/subdir$j/file.logfile
+		rm -rf rsyslog.input.dir$i/dir$j
+	done
+done
+
+# sleep a little to give rsyslog a chance for processing
+sleep 1
```

Comment:
any hard reason for this sleep?

Reply:
maybe deleted later. Test is not activated yet.